exporter/honeycombmarkerexporter/factory.go

```diff
@@ -36,12 +35,15 @@ func createLogsExporter(
 ) (exporter.Logs, error) {
 	cf := cfg.(*Config)
 
-	exporter := newLogsExporter(set.Logger, cf)
+	logsExp, err := newHoneycombLogsExporter(set.TelemetrySettings, cf)
+	if err != nil {
+		return nil, err
+	}
 
 	return exporterhelper.NewLogsExporter(
 		ctx,
 		set,
 		cfg,
-		exporter.exportLogs,
+		logsExp.exportMarkers,
```

Comment:
Since we need to setup our client we need to use the exporterhelper.WithStart(c.start) and exporterhelper.WithShutdown(c.stop)) options here.